controllers/status_controller.go

```diff
@@ -91,8 +92,15 @@ func (r *StatusReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctr
 		return ctrl.Result{}, retryErr
 	}
 
+	log.Info("updating socket")
```

Comment:
IMHO it would be a good idea to save the dedicated socket into a new field in order to preserve the information where the database still gets exposed.

Reply:
Now that it is implemented, I agree. In the migration phase, the dedicated loadbalancer will be indeed be an additional socket (and not replace the existing loadbalancer).